packages/react-instantsearch-core-next/src/__tests__/Index.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from '@testing-library/react';
+// eslint-disable-next-line import/named
+import { IndexWidget } from 'instantsearch.js/es/widgets/index/index';
+import React from 'react';
+
+import { createSearchClient } from '../../../../test/mock';
+import { IndexContext } from '../IndexContext';
+import { InstantSearch } from '../InstantSearch';
+import { Index } from '../SearchIndex';
+
+describe('Index', () => {
+  test('renders children', () => {
+    const searchClient = createSearchClient();
+
+    const { container } = render(
+      <InstantSearch indexName="indexName" searchClient={searchClient}>
```

Comment:
Do we still want to keep this pattern where we can provide an index name via a prop on `<InstantSearch>` or should we move to always using `<Index>`? Is it a requirement of InstantSearch Core to pass an `indexName`?

If there's a way around it, it could be nice to move to only using `Index` widgets in RIS. It's a lot clearer and makes a lot of sense with visually nested components.

Reply:
It is a requirement of InstantSearch Core, and it would be a consequent change.

Indeed, in the future it would be nice to remove this default index.

Reply:
Yes, and it makes sense to do it all at once in every InstantSearch libraries for consistency.

It isn't an emergency but we should keep this in mind. I opened a [ticket](https://algolia.atlassian.net/browse/FX-122) so we don't forget.

Reply:
it's interesting, but having a root index can be confusing, what results would eg. a hits on the root get? it would need logic added to make sure everything can render in some way without results as well, and a warning for the cases where you add no index at all

Reply:
The way I saw it, we'd drop `indexName` on the `<InstantSearch>` and have the same behavior by having a root `<Index>`. If you add a `<Hits>` widget not nested in an `<Index>` one it wouldn't work, just like right now you need to pass an `indexName`.

If you want to do multi-index you'd have multiple parallel `<Index>` widgets. You could imagine having authorized widgets outside of `<Index>` (e.g., `<Configure>`) that would be inherited, or overriden by local ones.